frontend/test/metabase-smoketest/admin.cy.spec.js

```diff
@@ -0,0 +1,381 @@
+import { USERS, restore, signOut } from "__support__/cypress";
+
+const admin = USERS.admin;
+const new_user = {
+  first_name: "Barb",
+  last_name: "Tabley",
+  username: "new@metabase.com",
+};
+
+describe("metabase-smoketest > admin", () => {
+  before(() => restore("blank"));
+
+  it("should set up Metabase", () => {
+    // ****************
+    // ***  ADMIN   ***
+    // *****************
+
+    // This is a simplified version of the "scenarios > setup" test
+    cy.visit("/");
+    cy.contains("Welcome to Metabase");
+    cy.url().should("not.include", "login");
+    cy.findByText("Let's get started").click();
+
+    // Language
+
+    cy.contains("What's your preferred language");
+    cy.findByText("English").click();
+    cy.findByText("Next").click();
+
+    // User (with workaround from "scenarios > setup"  document)
+
+    cy.contains("What should we call you?");
+
+    cy.findByLabelText("First name").type(admin.first_name);
+    cy.findByLabelText("Last name").type(admin.last_name);
+    cy.findByLabelText("Email").type(admin.username);
+    cy.findByLabelText("Your company or team name").type("Epic Team");
+
+    cy.findByLabelText("Create a password")
+      .clear()
+      .type(admin.password);
+    cy.findByLabelText("Confirm your password")
+      .clear()
+      .type(admin.password);
+    cy.findByText("Next").click();
+
+    // Database
+
+    cy.contains("Add your data");
+    cy.contains("I'll add my data later");
+
+    cy.findByText("Select a database").click();
+    cy.findByText("H2").click();
+    cy.findByLabelText("Name").type("Metabase H2");
+
+    const dbPath = path.resolve(
+      Cypress.config("fileServerFolder"),
+      "frontend/test/__runner__/empty.db",
+    );
+    cy.findByLabelText("Connection String").type(`file:${dbPath}`);
+    cy.findByText("Next").click();
+
+    // Turns off anonymous data collection
+    cy.findByLabelText(
+      "Allow Metabase to anonymously collect usage events",
+    ).click();
+    cy.findByText("All collection is completely anonymous.").should(
+      "not.exist",
+    );
+    cy.findByText("Next").click();
+
+    // Finish & Subscribe
+
+    cy.findByText("Take me to Metabase").click();
+    cy.url().should("be", "/");
+
+    // =================
+    // should add a simple summarized question
+    // =================
+
+    cy.contains("Bobby");
+    // This page does not contain "OUR DATA"
+    cy.contains("Our analytics", { timeout: 10000 });
+
+    // Following section is repeated-- turn into callback function?
+    // Also, selecting Metabase H2 doesn't do anything
+    cy.findByText("Ask a question").click();
+
+    cy.contains("Custom question");
+    cy.contains("Native query");
+
+    cy.findByText("Simple question").click();
+    cy.findByText("Sample Dataset").click();
+    cy.findByText("People").click();
+
+    cy.contains("Save");
+
+    // Filter for created within previous 5 years
+
+    cy.findByText("Filter").click();
+    cy.get(".scroll-y")
```

Comment:
I'm a little hesitant about the `.scroll-y` approach here, but not sure if @paulrosenzweig or @tlrobinson have a better approach for selecting from the sidebar.

Reply:
I'm pretty sure I've used that to grab the sidebar, but it does feel icky. If we don't want to start decorating our code with cypress selectors, I guess we could extract a `sidebar()` helper so that there's only one place to fix if using `.scroll-y` stops working.

Reply:
I could replace my use of `.scrolly-y` with a `cy.findAllByText("").last().click()` which (in most cases) would function the same way.

Reply:
Where would you want the `sidebar()` helper to be placed?